drivers/sensor/mchp_tach_xec/Kconfig

```diff
@@ -8,3 +8,28 @@ config TACH_XEC
 	depends on SOC_FAMILY_MEC
 	help
 	  Enable the Microchip XEC tachometer sensor.
+
+if TACH_XEC
+
+config TACH_XEC_9_TACH_EDGES
+	bool "Configure 9 tach edges or 4 tach periods"
+
+config TACH_XEC_5_TACH_EDGES
+	bool "Configure 5 tach edges or 2 tach periods"
+	default y
+
+config TACH_XEC_3_TACH_EDGES
+	bool "Configure 3 tach edges or 1 tach period"
+
+config TACH_XEC_2_TACH_EDGES
+	bool "Configure 2 tach edges or 1/2 tach period"
```

Comment:
These should be mutually exclusive, right? Put them in a choice

Reply:
i just wrapped the  possible option in a choice /endchoice